librocksdb-sys/src/lib.rs

```diff
@@ -17,6 +17,14 @@
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
 
+// Ensure the libraries are linked in, despite it not being used directly
```

Comment:
I'm not sure it works this way, but it looks good otherwise.

Reply:
I'm not sure if it is or not, I think this is what made me think it was necessary: https://kornel.ski/rust-sys-crate#linking.

Reply:
Commenting one out does cause linking to fail with unknown symbols, so I think it's needed